### PR TITLE
blog: March community gems

### PR DESCRIPTION
**Pull request by flippedcoder**
Tentative publish date: 03/29/22

Things left to update:
- [x] image (make sure to use https://tinypng.com/)
- [x] comments link
- [x] April office hours description/link
- [x] technical style (code blocks formatted correctly with dvc, python, yaml, etc)

### CODE REVIEW

File: content/blog/2022-03-29-march-22-community-gems.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+commentsUrl: https://discuss.dvc.org/t/march-22-community-gems/1119
+tags:
+  - Data Versioning
+  - DVC Remotes
+  - DVC API
+  - DVC Stages
+  - Community
+---
+
+### [What is the difference between using `dvc exp` run and `dvc repro`?](https://discord.com/channels/485586884165107732/485596304961962003/939070512322195456)
```
Comment (dberenbaum) [inline content/blog/2022-03-29-march-22-community-gems.md:21]:
```suggestion
### [What is the difference between using `dvc exp run` and `dvc repro`?](https://discord.com/channels/485586884165107732/485596304961962003/939070512322195456)
```

File: content/blog/2022-03-29-march-22-community-gems.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+This is a really good question from @v2.03.99!
+
+When you use `dvc exp run`, DVC automatically tracks each experiment run. Using
+`dvc repro` leaves it to the user to track each experiment.
```
Comment (dberenbaum) [inline content/blog/2022-03-29-march-22-community-gems.md:26]: Looks good! Consider adding a link for more info, like https://dvc.org/blog/experiment-refs or https://dvc.org/doc/user-guide/experiment-management/experiments-overview.

File: content/blog/2022-03-29-march-22-community-gems.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+A great question here from @quarkquark!
+
+You can debug in VSCode by following the steps below:
+
+- Install the `debugpy` package.
+- Navigate to `"Run and Debug" > "Remote Attach" > localhost > someport`.
+- In a terminal in VSCode,
+  `python -m debugpy --listen someport --wait-for-client -m dvc mycommand`
+
+This should help you debug the stages in your pipeline in the IDE.
```
Comment (dberenbaum) [inline content/blog/2022-03-29-march-22-community-gems.md:39]: Link for more info: https://github.com/iterative/dvc/wiki/Debugging-DVC-interactively.

File: content/blog/2022-03-29-march-22-community-gems.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+### [Is there a mailing list for subscribing to CML releases?](https://discord.com/channels/485586884165107732/728693131557732403/939215540591927337)
+
+It's awesome community members like @pria want to keep up with our releases!
+
+You can follow all of our releases via GitHub notifications. You can browse
+release notes at <https://github.com/iterative/cml/releases>. You can also
+subscribe to release updates by clicking the `Watch` button in the top-right,
+navigating to `Custom`, and checking the `Releases` option.
+
+![the checkbox you need to check in GitHub to follow CML releases](2022-03-29/cml_release_follow.png)
```
Comment (julieg18) [inline content/blog/2022-03-29-march-22-community-gems.md:79]: Images inside the blog posts need to be prefaced with `/uploads/images/`

```suggestion
![the checkbox you need to check in GitHub to follow CML releases](/uploads/images/2022-03-29/cml_release_follow.png)
```

**Comment by flippedcoder** [comment]
@dberenbaum Could I get an approval and merge for this today? 🙏🏽

**Review by julieg18 — APPROVED** [review]

**Pull request merged**